rrfs-test/IODA/yaml/prepbufr_aircft.yaml

```diff
@@ -31,112 +29,110 @@ observations:
             query: "*/YOB"
           stationIdentification:
             query: "*/SID"
-          aircraftFlightNumber:
-            query: "[*/ACID, */ACID_SEQ/ACID]"
-          prepbufrDataLevelCategory:
-            query: "*/PRSLEVLA/CAT"
+          stationElevation:
+            query: "*/ELV"
+            type: float
```

Comment:
similar here

Reply:
it seems to me for all height variables, an integer is enough.